content/en/blog/_posts/2024-04-02-introducing-jobset/2024-04-02-introducing-jobset.md

```diff
@@ -0,0 +1,261 @@
+essentially a Job Template with some desired number of Job replicas specified. This provides
+a declarative way to easily create identical child-jobs to run on different islands of
+accelerators, without resorting to scripting or Helm charts to generate many versions of the
+same job but with different names.
+
+![JobSet Architecture Diagram](jobset_diagram.svg "JobSet Architecture")
+
+Some other key JobSet features which address the problems described above include:
+
+Replicated Jobs
+: In modern data centers, hardware accelerators like GPUs and TPUs allocated
+in islands of homogenous accelerators connected via a specialized, high bandwidth network links.
+For example, a user might provision nodes containing a group of hosts co-located on a rack, 
+each with H100 GPUs, where GPU chips within each host are connected via NVLink, with a NVLink
+Switch connecting the multiple NVLinks. TPU Pods are another example of this: TPU ViperLitePods
+consist of 64 hosts, each with 4 TPU v5e chips attached, all connected via ICI mesh. When running
+a distributed training job across multiple of these islands, we often want to partition the
+workload into a group of smaller identical jobs, 1 per island, where each pod primarily
+communicates with the pods within the same island to do segments of distributed computation,
+and keeping the gradient synchronization over DCN (data center network, which is lower bandwidth
+than ICI) to a bare minimum. 
+
+Automatic headless service creation, configuration, and lifecycle management
+: Pod-to-pod communication via pod hostname is enabled by default, with automatic configuration
+and lifecycle management of the headless service enabling this. 
+
+Configurable success policies
+: JobSet has configurable success policies which target specific ReplicatedJobs, with operators
+to target “Any” or “All” of their child jobs. For example, you can configure the JobSet to be marked
+complete if and only if all pods that are part of the “worker” ReplicatedJob are completed.
+
+Configurable failure policies
+: JobSet has configurable failure policies which allow the user to specify a maximum number of
+times the JobSet should be restarted in the event of a failure. If any job is marked failed,
+the entire JobSet will be recreated, allowing the workload to resume from the last checkpoint.
+When no failure policy is specified, if any job fails, the JobSet simply fails. 
+
+Exclusive placement per topology domain
+: JobSet allows users to express that child jobs have 1:1 exclusive assignment to a topology
+domain, typically an accelerator island like a rack. For example, if the JobSet creates
+two child jobs, then this feature will enforce that the pods of each child job will be
+co-located on the same island, and that only one child job is allowed to schedule per island.
+This useful for scenarios where, for example, we want to use a data parallel distributed
```

Comment:
```suggestion
This is useful for scenarios where we want to use a data parallel distributed
```

Reply:
Updated with your suggestion but also rephrased to "distributed data parallel" to align with the PyTorch naming (see [DDP docs](https://pytorch.org/tutorials/intermediate/ddp_tutorial.html)).